packages/evm/src/fee/__test__/PercentageFee.test.ts

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
To remove

Reply:
Removal of these cause lint failures as I'm mocking some stuff for tests, and import some ethers js stuff using `require`